### ISSUE TITLE
v0.0.2 cannot import name 'PlotlyScope' on Windows venv

### ISSUE THREAD

**Issue by WEGFan** [plotly/Kaleido#22]
```python
>>> from kaleido.scopes.plotly import PlotlyScope
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "D:\My Documents\GitHub\codestats-profile-readme\venv\lib\site-packages\kaleido\scopes\plotly.py", line 4, in <module>
    from plotly.graph_objects import Figure
  File "D:\My Documents\GitHub\codestats-profile-readme\venv\lib\site-packages\plotly\__init__.py", line 34, in <module>
    from plotly import (
  File "D:\My Documents\GitHub\codestats-profile-readme\venv\lib\site-packages\plotly\io\__init__.py", line 5, in <module>
    from ._kaleido import to_image, write_image
  File "D:\My Documents\GitHub\codestats-profile-readme\venv\lib\site-packages\plotly\io\_kaleido.py", line 8, in <module>
    from kaleido.scopes.plotly import PlotlyScope
ImportError: cannot import name 'PlotlyScope'
```
It seems like it's a circular import problem and #18 causes it, but neither Linux with venv nor Windows without venv has this issue.
I'm using Python 3.6.8 on Windows and 3.6.2 on Linux.

**Comment by jonmmease** [comment]
Hi @WEGFan, what version of the `plotly` package do you have installed in each environment?

**Comment by WEGFan** [comment]
Both are 4.9.0

**Comment by WEGFan** [comment]
Oh I forgot to say I add a line in `plotly/io/_kaleido.py` here to find out what caused the `PlotlyScope`be `None`
```python
try:
    from kaleido.scopes.plotly import PlotlyScope

    scope = PlotlyScope()

    # Compute absolute path to the 'plotly/package_data/' directory
    root_dir = os.path.dirname(os.path.abspath(plotly.__file__))
    package_dir = os.path.join(root_dir, "package_data")
    scope.plotlyjs = os.path.join(package_dir, "plotly.min.js")
    scope.mathjax = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js"
except ImportError:
    raise  # here
    PlotlyScope = None
    scope = None
```

**Comment by jonmmease** [comment]
Thanks for info the debugging @WEGFan. I'm not able to reproduce yet, but there is a circular import here that we might as well get rid of.

In `kaleido/py/kaleido/scopes/plotly.py`, we can move the `from plotly.graph_objects import Figure` related import down into the `transform` method. 

Here is a PR that does this: https://github.com/plotly/Kaleido/pull/25.  Could you give this a try (The change is moving a single line, so you could edit it manually as you did above).

**Comment by WEGFan** [comment]
It worked, but I still wonder why it happened only in venv...

**Comment by jonmmease** [comment]
Great! I'm also not sure what is going on with the virtual environment there.

Until we get the next release out, I'm pretty sure you wouldn't run into this issue if you could use Python 3.7+ in your venv since plotly.py imports submodules, like `plotly.io`, lazily in this case.

**Comment by jonmmease** [comment]
This fix has been released in version 0.0.3
